syntax/odin.vim

```diff
@@ -4,21 +4,23 @@ vim9script
 # Language: Odin
 # Maintainer: Maxim Kim <habamax@gmail.com>
 # Website: https://github.com/habamax/vim-odin
-# Last Change: 2024-01-15
+# Last Change: 2025-03-28
 
 if exists("b:current_syntax")
   finish
 endif
 
-syntax keyword odinKeyword using transmute cast distinct opaque where dynamic
+syntax keyword odinKeyword using transmute cast auto_cast distinct opaque where dynamic
 syntax keyword odinKeyword struct enum union const bit_field bit_set
 syntax keyword odinKeyword package proc map import export foreign
 syntax keyword odinKeyword size_of offset_of type_info_of typeid_of type_of align_of
 syntax keyword odinKeyword return defer
 syntax keyword odinKeyword or_return or_else
 syntax keyword odinKeyword inline no_inline
+syntax keyword odinKeyword asm context
 
-syntax keyword odinConditional if when else do for switch case continue break
+syntax keyword odinConditional if when else do for switch case fallthrough
+syntax keyword odinConditional continue or_continue break or_break
```

Comment:
shouldn't or_else and or_break be in the same category? Either keyword or conditional

Reply:
Yea... I put these ones with the existing `continue` and `break` and didn't want to change the existing `or_return` / `or_else`.

It does strike me as odd that they aren't categorized the same, and _maybe_ it makes more sense for `or_return` (and even `return`) to be categorized as `conditional` (really, `control-flow` imo). Ehhhh.

It doesn't really matter to me though, all of these are linked to the same group (statement) anyway. I'll leave it up to you.